scripts/exregional_run_fcst.sh

```diff
@@ -407,6 +408,15 @@ else
   fi
 fi
 
+#
+if [ "${CCPP_PHYS_SUITE}" = "FV3_HRRR_smoke" ]; then
+ ln -sf  /mnt/lfs4/BMC/gsd-fv3-dev/Haiqin.Li/ForcingData/NA_3km/dust_3km_test.nc  ${run_dir}/INPUT/dust12m_data.nc
+ ln -sf  /mnt/lfs4/BMC/gsd-fv3-dev/Haiqin.Li/ForcingData/NA_3km/antro_3km_in.nc  ${run_dir}/INPUT/emi_data.nc
+ yyyymmdd=${cdate:0:8}
+ ln -sf  /mnt/lfs4/BMC/gsd-fv3-dev/Haiqin.Li/ForcingData/NA_3km/FIRE_GBBEPx_data_3km_in.nc   ${run_dir}/INPUT/SMOKE_GBBEPx_data.nc
+#ln -sf  /mnt/lfs4/BMC/gsd-fv3-dev/Haiqin.Li/ForcingData/NA_3km/${yyyymmdd}/FIRE_GBBEPx_data_3km_in.nc   ${run_dir}/INPUT/SMOKE_GBBEPx_data.nc
+fi
```

Comment:
Are the above nc files fixed files? If yes, we will need to put them under the fix/ directory and I can help you with that.

Reply:
Thanks, Guoqing. The dust12m_data.nc and emi_data.nc are fixed files. The wildfire input data SMOKE_GBBEPx_data.nc is a time varying input for each forecast day.

Reply:
@haiqinli   (and tag @hu5970 )
I made some changes and created a PR to your branch. You can review it at:
https://github.com/haiqinli/regional_workflow/pull/1

If it is fine with you, you can merge the above PR to your branch and then do some tests to make sure it works as expected.

Thanks!

Reply:
How do you generate "SMOKE_GBBEPx_data.nc" file? Or it is provided as "observation" by other people?
The location of this file need to be set in configure so the system can be ported to run on other HPCs.